public/javascripts/data/signup.js

```diff
@@ -0,0 +1,33 @@
+const Database = require('better-sqlite3');
```

Comment:
Combine your `signup.js` and `login.js` files into one - `user.js`. The data object (the user) is the same one being manipulated for login & signup, so it makes sense from a code organization standpoint to keep them in the same file.